src/rules/noInternalModuleRule.ts

```diff
@@ -19,7 +19,7 @@ import * as ts from "typescript";
 import * as Lint from "../lint";
 
 export class Rule extends Lint.Rules.AbstractRule {
-    public static FAILURE_STRING = "forbidden internal module";
+    public static FAILURE_STRING = "Internal modules are forbidden. Use ES6 module syntax instead.";
```

Comment:
This message isn't quite accurate: this rule encourages the use of `namespace` instead of `module` for creating a namespace structure in TS. (The `no-namespace` rule is used for what you were thinking of.) Instead, this error message should say something about using the `namespace` keyword instead.